bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,228 @@
+import logging
+import random
+import typing
+from json import load
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import Colours, Lovefest, Roles
+from bot.decorators import with_role
+
+log = logging.getLogger(__name__)
+
+HEART_EMOJIS = [":heart:", ":gift_heart:", ":revolving_hearts:", ":sparkling_heart:", ":two_hearts:"]
+EMOJI_1 = random.choice(HEART_EMOJIS)
```

Comment:
If these emojis are defined at the module level they're never going to change unless the cog is reloaded. If you want to have them cycling for each valentine then it would probably be more appropriate to have a static method that returns a random emoji from the list.